v2/googlecloud-to-elasticsearch/src/main/java/com/google/cloud/teleport/v2/elasticsearch/options/PubSubToElasticsearchOptions.java

```diff
@@ -62,4 +62,10 @@
   String getDeadletterTable();
```

Comment:
Based on recent inclusive language standards Deadletter is a disrespectful word.  Instead use the option name as errorOutputTable  (i.e getErrorOutputTable, setErrorOutputTable).  Please update docs as well